lib/advisor/web/authentication/password.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Advisor.Web.Authentication.Password do
+  alias Comeonin.Bcrypt
+  def matches?(password) do
+    [checker: checker, password: real_hash] = Application.get_env(:advisor, __MODULE__)
+
+    checker.matches?(password, real_hash)
```

Comment:
What do you think to call `real_hash` -> `hashed_password`
I mean, I am not sure `real` give really any information as there aren't any `fake_hash` :D